createbillboardplaylist.py

```diff
@@ -311,6 +311,14 @@ def create_all(self):
             "http://www.billboard.com/charts/hot-100",
         )
 
+		# Billboard Country 50
+        self.create_playlist_from_chart(
+            "country-50",
```

Comment:
The ID for a chart is the same as the last segment of the URL on the billboard.com website.

```suggestion
            "country-songs",
```